LFGBulletinBoard/LibGPIToolBox.lua

```diff
@@ -1,8 +1,10 @@
 local TOCNAME,
 	---@class Addon_Tool	
 	Addon = ...;
-Addon.Tool=Addon.Tool or {}
-local Tool=Addon.Tool
+
+---@class ToolBox
+local Tool = {}
+Addon.Tool = Tool
 
```

Comment:
Order here doesnt matter. Could defined `Tool` in terms of `Addon.Tool` but doing it this way to keep LuaLS happy.